### ISSUE TITLE
Create "information pack" about our app for Wikimedia chapters who are interested in running Commons workshops

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#3374]
It feels to me like our app could improve our engagement with Wikimedia chapters - currently, AFAIK, the only chapter that is familiar with us is WMCZ. :) In a potential future grant, I was thinking about contacting other chapters about our app and creating an "information pack" that we could offer to those who are interested. This could contain perhaps a presentation about our app with screenshots, download links, a brief "getting started" tutorial, etc. 

The chapters could use this if they were interested in, for instance, including our app in their Commons editing workshops.

@VojtechDostal @sivaraam What do you guys think?

**Comment by VojtechDostal** [comment]
Yeah, I think this would be very useful for some chapters who work in this field :-) 

I will also reach out to our multimedia manager Jakub if he has any ideas on this.
